src/derivative_operators/derivative_operator_functions.jl

```diff
@@ -115,7 +115,7 @@ function *(A::DerivativeOperator{T,N},M::AbstractArray{T}) where {T<:Real,N}
 end
 
 function *(c::Number, A::DerivativeOperator{T,N,Wind}) where {T,N,Wind}
-    coefficients = A.coefficients === nothing ? one(T) .* c : c .* A.coefficients
+    coefficients = A.coefficients === nothing ? c : c .* A.coefficients
```

Comment:
```suggestion
    coefficients = A.coefficients === nothing ? oneunit(T) .* c : c .* A.coefficients
```

I missed what you were doing here the first time.

Reply:
Though if it defaulted to a FillVector we would remove an allocation. We should make sure an issue gets opened on this.

Reply:
I believe we don't need to make the coefficient a vector, we can just convert it to type `T`. For example, run the following example (some version of which should be in the tests):
`L = CenteredDifference(1, 2, 1., 5)`
`c = 3.`
`x = rand(7)`
`cL = c * L`
`@assert length(cL.coefficients) == 1`
`cL * x == c * (L * x)`